install_config/topics/ocp_osp_deploy_instance.adoc

```diff
@@ -0,0 +1,275 @@
+----
+
+
+Once registered, enable the following repositories as follows.
+
+[subs=+quotes]
+----
+$ sudo subscription-manager repos \
+    --enable="rhel-7-server-rpms" \
+    --enable="rhel-7-server-extras-rpms" \
+    --enable="rhel-7-server-ose-3.10-rpms" \
+    --enable="rhel-7-fast-datapath-rpms" \
+    --enable="rhel-7-server-ansible-2.5-rpms" \
+    --enable="rhel-7-server-openstack-13-rpms" \
+    --enable="rhel-7-server-openstack-13-tools-rpms"
+----
+
+====== Required Packages on the Deployment Host
+
+The following packages are required to be installed on the deployment host.
+
+Install the following packages:
+
+* `openshift-ansible`
+* `python-openstackclient`
+* `python2-heatclient`
+* `python2-octaviaclient`
+* `python2-shade`
+* `python-dns`
+* `git`
+* `ansible`
+
+[subs=+quotes]
+----
+$ sudo yum -y install openshift-ansible python-openstackclient python2-heatclient python2-octaviaclient python2-shade python-dns git ansible
+----
+
+====== Configure Ansible
+
+`ansible` is installed on the deployment instance to perform the registration, 
+installation of packages, and the deployment of the {product-title} environment on the 
+master and node instances.
+
+Before running playbooks, it is important to create a _ansible.cfg_ to reflect 
+the environment we wish to deploy:
+
+[subs=+quotes]
+----
+$ *cat ~/ansible.cfg*
+
+[defaults]
+forks = 20
+host_key_checking = False
+remote_user = openshift
+gathering = smart
+fact_caching = jsonfile
+fact_caching_connection = $HOME/ansible/facts
+fact_caching_timeout = 600
+log_path = $HOME/ansible.log
+nocows = 1
+callback_whitelist = profile_tasks
+inventory = /usr/share/ansible/openshift-ansible/playbooks/openstack/inventory.py,/home/cloud-user/inventory
+
+[ssh_connection]
+ssh_args = -o ControlMaster=auto -o ControlPersist=600s -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=false
+control_path = %(directory)s/%%h-%%r
+pipelining = True
+timeout = 10
+
+[persistent_connection]
+connect_timeout = 30
+connect_retries = 30
+connect_interval = 1
+----
+
+[WARNING]
+====
+
+The following parameters values are important to the ansible.cfg file.
+
+* The `remote_user` must remain as the user _openshift_. 
+* The inventory parameter ensure that there is no space between the two
+inventories.
+
+Example: _inventory = path/to/inventory1,path/to/inventory2_
+====
+
+NOTE: The code block above can overwrite the default values in the file. 
+Ensure to populate <keypair-name> with the keypair that was copied to the 
+deployment instance.
+
+NOTE: The _inventory_ folder is created in <<inventory_provision>>. 
+
+====== OpenShift Authentication
+
+{product-title} provides the ability to use many different authentication platforms.
+A listing of authentication options are available at
+https://docs.openshift.com/container-platform/3.10/install_config/configuring_authentication.html#install-config-configuring-authentication[Configuring Authentication and User Agent].
```

Comment:
xref:../install_config/configuring_authentication.adoc#install-config-configuring-authentication[Configuring Authentication and User Agent].